cli/src/cli.rs

```diff
@@ -120,8 +122,40 @@ pub enum Subcommand {
 }
 
 #[derive(Debug, StructOpt)]
-pub struct DvmParameters {
+pub struct DvmArgs {
+	/// Enable EVM tracing module on a non-authority node.
+	#[structopt(long, conflicts_with = "validator", require_delimiter = true)]
+	pub ethapi: Vec<EthApiCmd>,
+
+	/// Number of concurrent tracing tasks. Meant to be shared by both "debug" and "trace" modules.
+	#[structopt(long, default_value = "10")]
+	pub ethapi_max_permits: u32,
+
+	/// Maximum number of trace entries a single request of `trace_filter` is allowed to return.
+	/// A request asking for more or an unbounded one going over this limit will both return an
+	/// error.
+	#[structopt(long, default_value = "500")]
+	pub ethapi_trace_max_count: u32,
+
+	/// Duration (in seconds) after which the cache of `trace_filter` for a given block will be
+	/// discarded.
+	#[structopt(long, default_value = "300")]
+	pub ethapi_trace_cache_duration: u64,
+
+	/// Size of the LRU cache for block data and their transaction statuses.
+	#[structopt(long, default_value = "3000")]
+	pub eth_log_block_cache: usize,
+
 	/// Maximum number of logs in a query.
 	#[structopt(long, default_value = "10000")]
 	pub max_past_logs: u32,
+
+	/// Choose sealing method.
+	#[cfg(feature = "template")]
+	#[structopt(long = "sealing", default_value)]
+	pub sealing: Sealing,
+
+	#[cfg(feature = "template")]
+	#[structopt(long = "enable-dev-signer")]
+	pub enable_dev_signer: bool,
```

Comment:
`sealing` and `enable_dev_signer` are only used in the template manual seal consensus, we don't need them here.